test/integration/smoke/test_outofbandmanagement.py

```diff
@@ -0,0 +1,561 @@
+        self.assertEqual(response.powerstate, 'On')
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_issue_power_on(self):
+        """
+            Tests out-of-band management issue power on action
+        """
+        self.configureAndEnableOobm()
+        self.startIpmiServer()
+        self.issuePowerActionCmd('ON')
+        response = self.issuePowerActionCmd('STATUS')
+        self.assertEqual(response.powerstate, 'On')
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_issue_power_off(self):
+        """
+            Tests out-of-band management issue power off action
+        """
+        self.configureAndEnableOobm()
+        self.startIpmiServer()
+        self.issuePowerActionCmd('OFF')
+        response = self.issuePowerActionCmd('STATUS')
+        self.assertEqual(response.powerstate, 'Off')
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_issue_power_cycle(self):
+        """
+            Tests out-of-band management issue power cycle action
+        """
+        self.configureAndEnableOobm()
+        self.startIpmiServer()
+        self.issuePowerActionCmd('CYCLE')
+        response = self.issuePowerActionCmd('STATUS')
+        self.assertEqual(response.powerstate, 'On')
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_issue_power_reset(self):
+        """
+            Tests out-of-band management issue power reset action
+        """
+        self.configureAndEnableOobm()
+        self.startIpmiServer()
+        self.issuePowerActionCmd('RESET')
+        response = self.issuePowerActionCmd('STATUS')
+        self.assertEqual(response.powerstate, 'On')
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_issue_power_soft(self):
+        """
+            Tests out-of-band management issue power soft action
+        """
+        self.configureAndEnableOobm()
+        self.startIpmiServer()
+        self.issuePowerActionCmd('SOFT')
+        response = self.issuePowerActionCmd('STATUS')
+        self.assertEqual(response.powerstate, 'Off')
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_background_powerstate_sync(self):
+        """
+            Tests out-of-band management background powerstate sync
+        """
+        self.debug("Testing oobm background sync")
+        interval = list_configurations(
+            self.apiclient,
+            name='outofbandmanagement.sync.interval'
+        )[0].value
+
+        self.configureAndEnableOobm()
+        self.startIpmiServer()
+        bmc = IpmiServerContext().bmc
+
+        bmc.powerstate = 'on'
+        self.checkSyncToState('On', interval)
+
+        bmc.powerstate = 'off'
+        self.checkSyncToState('Off', interval)
+
+        self.server.shutdown()
+        self.server.server_close()
+
+        # Check for unknown state (ipmi server not reachable)
+        self.checkSyncToState('Unknown', interval)
+
+
+    @attr(tags=["smoke", "advanced"])
+    def test_oobm_multiple_mgmt_server_ownership(self):
```

Comment:
This is what this test does, by simulating another mgmt server by doing a direct db manipulation and checks how the current mgmgt server behaves.